pyasic/miners/backends/braiins_os.py

```diff
@@ -1058,3 +1064,28 @@ async def _get_uptime(self, rpc_summary: dict = None) -> Optional[int]:
                 return int(rpc_summary["SUMMARY"][0]["Elapsed"])
             except LookupError:
                 pass
+
+    async def _get_pools(channel: Channel, grpc_pools: dict = None) -> List[PoolMetrics]:
+        if grpc_pools is None:
+            try:
+                grpc_pools = await _get_pools_Group(channel)
```

Comment:
Use this function instead - https://github.com/UpstreamData/pyasic/blob/30f385c2d9513605bb3c46ed7e974f622449f955/pyasic/web/braiins_os/boser.py#L412-L415

Reply:
Okay, Noted.